api-reference/v1.0/toc.yml

```
@@ -5043,6 +5045,20 @@ items:
                   href: api/ipnamedlocation-update.md
                 - name: Delete
                   href: api/ipnamedlocation-delete.md
+          - name: Authentication context
+            items:
+            - name: Authentication context class reference
+              href: resources/authenticationcontextclassreference.md
+            - name: List
+              href: api/conditionalaccessroot-list-authenticationcontextclassreferences.md
+            - name: Create
+              href: api/conditionalaccessroot-patch-authenticationcontextclassreferences.md
```

Following the file naming guidance, Create operations have the syntax: {parentEntityName}-post-{navigationPropertyName}.md. Therefore, **conditionalaccessroot-post-authenticationcontextclassreferences.md**

PATCH is handled by the -update.md file.
```suggestion
              href: api/conditionalaccessroot-post-authenticationcontextclassreferences.md
```